packages/hsl-shared-components/src/TextInput/Text.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { lighten } from 'polished';
+import { size } from '../utils';
+
+const Input = styled.input `
+  font-size: ${size(18)};
+  ${props => props.theme.fontFamily && `font-family: ${props.theme.fontFamily};`}
+  color: ${props => props.theme.secondary};
+
+  padding: ${size(14)} ${size(18)};
```

Comment:
I'm not sure, but to me it looks like there's just a bit too much padding here compared to UIkit. 
Maybe try something like `padding: ${size(14)} ${size(18)};`

Reply:
![1](https://user-images.githubusercontent.com/1543527/30471340-12106e2c-9a01-11e7-9dc5-2ff59acf6cff.png)

Reply:
When comparing images, then UI-kit seems to have less padding and larger font.
I'll reduce padding and increase font size slightly